pyproject.toml

```diff
@@ -7,12 +7,12 @@ version = "0.1.0"
 [tool.poetry.dependencies]
 pandas= "1.2.3"
```

Comment:
why is pandas the only dependency with a fixed version instead of a minimum one like the others?

Reply:
The version pinning wasn't related to this PR, but to the previous way in which the module was created. For this specific case, you can remove the version pin.